source/text/typesetter.ts

```diff
@@ -283,6 +292,43 @@ export class Typesetter {
 
                 feedVertexIndex = vertexIndex;
 
+            } else if (elideRemainingGlyphs) {
+                /** replace last 2 glyphs with .. */
+
+                /* roll back last 2 glyphs */
+                --index;
+                let precedingGlyph = label.fontFace!.glyph(label.charCodeAt(index));
+                pen[0] -= precedingGlyph.advance;
+                pen[0] -= label.kerningBefore(index);
+                --index;
+                precedingGlyph = label.fontFace!.glyph(label.charCodeAt(index));
+                pen[0] -= precedingGlyph.advance;
+                pen[0] -= label.kerningBefore(index);
+
+                vertexIndex -= 2;
+
+                /* update text */
+                const newText = label.text;
+                newText.text = newText.text.slice(0, index);
+                newText.text += '..';
```

Comment:
provide the elide symbol/string as variable (the width of which needs to be removed from the maxLineWidth.